### ISSUE TITLE
re-labeling of distortions and some humans to undefined

### ISSUE THREAD

**Issue by SopsyHallow** [retcons/limbus-logs#7]
on the site a lot of distortions and human enemies still have a difficulty rating and an ID tag, but most of them ingame have been changed to Undefined;Undefined (not all, however the only ones ive seen that still have it are EGO enemies)

I havent checked all, but heres a few that i have seen for sure

Bongy (both chicken and papa)
Bamboo Hatted Kim
Dongrang, who denies all
Kromer (both forms)

I will say that i am unsure if the undefined rating is properly backported ingame, as all of the ones i have mentioned are from MD4, which is their latest appearance and made after the announcement of reclassification. i wouldn't be supprised if the in-story versions still have classifications whether because it was missed in the change, or because it was deliberatly left there (the reclasification has a mini-sode, meanign that its probably an in-universe thing, which would mean that backporting properly would not fit the story)
I do personally think that the latest version should be used, since that is most accurate and up to date, but i suppose you can disagree with that assesment

**Comment by retcons** [comment]
yeah i noticed this a while back, i agree with the sentiment that the latest versions of classifications should be used

ill close this when i get around to going back and fixing all that. thank you!
